src/Algebra/Properties/Monoid.agda

```diff
@@ -0,0 +1,30 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some theory for Monoid
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Algebra using (Monoid)
+import Algebra.Properties.Semigroup
+open import Data.Product using (_,_)
+open import Relation.Binary using (Reflexive)
+
+module Algebra.Properties.Monoid {a ℓ} (M : Monoid a ℓ) where
+
+open Monoid M
+open Algebra.Properties.Semigroup semigroup public
+
+------------------------------------------------------------------------------
+-- Properties of divisibility
+------------------------------------------------------------------------------
+
+ε∣ : ∀ x → ε ∣ x
+ε∣ x = (x , sym (identityʳ x))
```

Comment:
Again unnecessary brackets here and below.

Reply:
Done.